### PR TITLE
[rllib] support local mode

### PR DESCRIPTION
**Pull request by ericl**
https://github.com/ray-project/ray/issues/2787#issuecomment-417770811

### CODE REVIEW

**Comment by robertnishihara** [comment]
Nice! Can we add a small test for this also?

**Comment by ericl** [comment]
Pushed.

File: python/ray/rllib/test/test_local.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from ray.rllib.agents.ppo import PPOAgent, DEFAULT_CONFIG
+import ray
+
+
+class LocalModeTest(unittest.TestCase):
+    def testLocal(self):
+        ray.init(local_mode=True)
+        cf = DEFAULT_CONFIG.copy()
+        agent = PPOAgent(cf, "CartPole-v0")
+        print(agent.train())
```
Comment (richardliaw) [inline python/ray/rllib/test/test_local.py]: Doesn't Jenkins run in 2.7...?

Comment (robertnishihara) [reply python/ray/rllib/test/test_local.py]: What's wrong?

Comment (richardliaw) [reply python/ray/rllib/test/test_local.py]: Oh the print() statement should fail, but you got the fix.

**Review by robertnishihara — APPROVED** [review]

**Pull request merged**